.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,3 +10,6 @@ PR Checklist
  - [ ] Docs?
  - [ ] CHANGELOG updated?
  - [ ] Issue raised/referenced?
+
+## Developers certificate of origin
+- [ ] I certify that this contribution is covered by the LGPLv2.1+ license as defined [here](https://github.com/MDAnalysis/mdanalysis/blob/develop/LICENSE) and adheres to the [**Developer Certificate of Origin**](https://developercertificate.org/).
```

Comment:
Does thinking this box from the UI register as an "edit"?

Reply:
Yes, you can see an example of the edit history from me ticking the test checklist in this PR.

Reply:
Thanks! Was on mobile, so I didn't see.

Reply:
I should have said "you can now see" - I only ticked it after you asked 🙃